FacuReartes/react-laba-final-project-team-2
# SPT2-127 - HomePage - Fetch update

# Description

Update Filter fetching and Home Page responsiveness

## Changes

- Updated Home Page filter fetching
- Refactored filter fields for improved readability.
- Products responsiveness in Home Page.
- Updated filter fields to match project requirements.

### Jira

- [SPT2-127](https://solvd-final-project-team-2.atlassian.net/browse/SPT2-127?atlOrigin=eyJpIjoiYzhhZTVmYjFiOTExNGJmMjliYmRiODcxZWE3Zjc4NWQiLCJwIjoiaiJ9)
- [SPT2-137](https://solvd-final-project-team-2.atlassian.net/browse/SPT2-137?atlOrigin=eyJpIjoiYWZkZDAzMjc4NTM0NGVkNjhjNjA1ZjRmOWU4Yjg3YWUiLCJwIjoiaiJ9)
- [SPT2-138](https://solvd-final-project-team-2.atlassian.net/browse/SPT2-138?atlOrigin=eyJpIjoiZmQxNjJiNmZiMTgxNDYyYjgyNGViOTA4NTMwOWNmYzkiLCJwIjoiaiJ9)

### How to Test

1. Navigate to ['http://localhost:3000/'](http://localhost:3000/).
2. Verify filter in Show Filters button next to 'Search Results'.
3. Select the filter options.
4. Other test is to change params in url to see filtering work

### Checklist


[SPT2-127]: https://solvd-final-project-team-2.atlassian.net/browse/SPT2-127?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[SPT2-137]: https://solvd-final-project-team-2.atlassian.net/browse/SPT2-137?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[SPT2-138]: https://solvd-final-project-team-2.atlassian.net/browse/SPT2-138?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
